packages/fxa-payments-server/src/lib/hooks.tsx

```diff
@@ -67,3 +69,11 @@ export function useClickOutsideEffect<T>(onClickOutside: Function) {
 
   return insideEl;
 }
+
+export function useNonce(
+  generateNonce = () => `${Date.now()}-${Math.random()}`
```

Comment:
Why not just use a uuid4? (I don't know how large Math.random is offhand, but it seems less random than a uuid4? )

Reply:
🤷‍♂ This seemed random enough for a single user hammering on the button. Doesn't currently need to be a univerally unique value